tests/src/JIT/opt/Enum/hasflag.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Some simple tests for the Enum.HasFlag optimization.
+// All the calls to HasFlag should be optimized to simple compares.
```

Comment:
And are they all optimized with this code, or is there more to do for some of them?

Reply:
All are now optimized (initially the ones with calls weren't). I'll fix the comment to be clearer.